Gemfile

```diff
@@ -6,6 +6,7 @@ group :development, :unit_tests do
   gem 'rspec-puppet',                  :require => false
   gem 'puppetlabs_spec_helper',        :require => false
   gem 'puppet-lint',                   :require => false
+  gem 'metadata-json-lint',            :require => false
```

Comment:
Why did you add the gem? If this is a needed dependency for one of the existing gems, than this should be addresses in a separate commit.

Reply:
It’s a common mistake to remove the ‘dependencies’ key from the metadata if there are no dependencies. But this would make the metadata invalid, (if there are no dependencies, you still need an empty array). This gem guards against this from happening in the future.